src/ui/AuthorAndTimestampLabel.tsx

```diff
@@ -65,18 +71,6 @@ function AuthorAndTimestampLabel({
 					</Label>
 				</Flex>
 			)}
-
-			{timestampLabel && (
```

Comment:
Timestamp label will not be shown if the display name is available. For example, when we are adding a comment.

Reply:
Ignore this comment.